src/cli/README.md

```diff
@@ -1543,6 +1544,84 @@ Set the ROUTER_UPGRADE_THRESHOLD value.
 Done
 ```
 
+### rssfilter
+
+Get current rss filter settings.
+
+```bash
+> rssfilter
+Default rss: -95 (lqi 1)
+dead00beef00cafe  * fixed rss -60 (lqi 3)
+dead00beef00caff  * fixed rss -95 (lqi 1)
+Done
+```
+
+### rssfilter set-lqi \<lqi\>
+
+Set default link quality in for all received messages
+
+```bash
+> rssfilter set-lqi 1
+Done
+```
+
+### rssfilter set-rss \<rss\>
+Set default rss in for all received messages
+
+```bash
+> rssfilter set-rss -95
+Done
+```
+
+### rssfilter get
+Get default rss/link quality in in for all received messages
+
+```bash
+> rssfilter get
+Default rss -95 (lqi 1)
+Done
+```
+
+### rssfilter unset
+Unset default rss/link quality in in for all received messages
+equivalent with cmd 'rssfilter set-rss 127'
+
+```bash
+> rssfilter unset
+Done
+```
+
+### rssfilter add-lqi \<extaddr\> \<lqi\>
+Set link quality in for the received messages from specified address
+
+```bash
+> rssfilter add-lqi dead00beef00caff 1
+Done
+```
+
+### rssfilter add-rss \<extaddr\> \<rss\>
```

Comment:
How about using "*" for `extaddr` to update the default received signal strength?  Then we can remove the separate CLI calls above.